src/math/Interpolate.cc

```diff
@@ -161,6 +161,9 @@ ::gsl_interp_type const *styleToGslInterpType(Interpolate::Style const style) {
             throw LSST_EXCEPT(pex::exceptions::LogicError,
                               str(boost::format("You can't get here: style == %") % style));
     }
+    // don't use default statement to let compiler check switch coverage
+    throw LSST_EXCEPT(pex::exceptions::LogicError,
+                      str(boost::format("You can't get here: style == %") % style));
```

Comment:
You can get "a proper stack trace" from the core dump, or from re-running under a debugger. Throwing an exception does _not_ guarantee that you're going to get a stack trace at all, or ever even notice that there's something horribly wrong.

It's absolutely essential when we _know_ that the system is in an inconsistent state (e.g., smashed stack, memory corruption) that we not attempt to push forward and make things worse and undebuggable.